cms_delete_content/templates/delete_confirm.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template id="delete_confirm" name="CMS default delete confirm">
```

Comment:
Good to know :) Anyway, that template is already loaded on demand by the ajax call and I prefer to keep the logic in the controller (+mixin) rather than in the JS.